docs/UserGuide.md

```diff
@@ -95,39 +95,82 @@ Format: `unmark INDEX`
+Format: `edit INDEX [n/NAME] [d/DESCRIPTION] [dl/DEADLINE] [p/PRIORITY] [t/TAG]...`
+- `DEADLINE` should be in the format `YYYY-MM-DD`.
+- `PRIORITY` should be one of `low`, `medium`, `high`.
+- `TAG` should not contain any space.
 
-Example: `edit 2 d/Prepare for CS2103T Tutorial` Edits the description of the 2nd task to be `Prepare for CS2103T Tutorial`
+Examples:
+- `edit 1 d/Prepare for CS2103T tutorial`
+- `edit 2 n/CS2103T meeting t/CS2103T`
 
-### Exiting the program : `exit`
+<div markdown="span" class="alert alert-primary">:bulb: **Note:**
+You can key `list` to see the sequence of tasks in the list to check the index of the task you wish to edit.
+</div>
+
+### 3.9 [Coming soon] Sorting tasks : `sort`
+
+Sorts the tasks by the specified sort key and sort order and list them out.
+
+Format: `sort by/SORT_KEY in/SORT_ORDER`
+- `SORT_KEY` is the property of the task used for sorting. Currently, supported sort keys are:
+  - `deadline` - sort by the due date of the task
+  - `priority` - sort by the priority of the task
+  - `name` - sort by the name of the task
+- `SORT_ORDER` is the order in which the tasks are listed out.
+  - `ascending`
+  - `descending`
+  
+Example:
+- `sort by/deadline in/ascending`: lists the tasks with the earliest deadline first
+- `sort by/priority in/descending`: lists the tasks with the highest priority first
+
+<div markdown="span" class="alert alert-primary">:bulb: **Note:**
+You can use the following abbreviations to sort tasks faster.
+- `asc` in place of `ascending`
+- `desc` in place of `descending`
+</div>
+
+### 3.10 Viewing help: `help`
+
+Shows the link to the user guide.
+
+Format: `help`
+
+### 3.11 Exiting the program : `exit`
 
 Exits the program.
 
 Format: `exit`
 
-### Saving the data
+### 3.12 Saving the data
 
 Harmonia's data is saved in the hard disk automatically after any command changes the data. There is no need to save manually.
 
+### 3.13 Editing the data file
+
+Harmonia's data is saved as JSON file ([Location of Harmonia.jar]/data/harmonia.json). Advanced users are encouraged to update the data by editing JSON file.
+
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
+If your changes to the data file makes its format invalid, Harmonia will discard all data and start with an empty data file at the next run.
+</div>
+
 --------------------------------------------------------------------------------------------------------------------
 ## Command summary
 
 Action | Format, Examples
 --------|------------------
-**Add** | `add d/DESCRIPTION` <br> e.g., `add d/read book`
+**Add** | `add n/NAME d/DESCRIPTION dl/DEADLINE p/PRIORITY [t/TAG]…` <br> e.g., `add n/CS2103T tp meeting d/read the weekly tasks before the meeting dl/2022-03-27 p/medium t/CS2103T t/meeting`
 **Delete** | `delete INDEX`<br> e.g., `delete 3`
 **Edit** | `edit INDEX [d/DESCRIPTION] [t/TAG]`<br> e.g.,`edit 2 n/CS2101 meeting t/CS2101`
```

Comment:
I think this should also be updated to `edit INDEX [n/NAME] [d/DESCRIPTION] [dl/DEADLINE] [p/PRIORITY] [t/TAG]...`

Reply:
Good catch, will update it!